Gemfile.lock

```diff
@@ -155,6 +155,8 @@ GEM
       trainer
       xcodeproj
       xctest_list (>= 1.2.1)
+    fastlane-plugin-trainer (0.4.1)
```

Comment:
There was some discussion on slack about why this happens, but there should not be changes in the Gemfile.lock in the diff

Reply:
Fixed with https://github.com/woocommerce/woocommerce-ios/pull/4441/commits/2a2e667bd9c708bb6491987a34ee069199101522